src/style.js

```diff
@@ -12,6 +12,10 @@ const style = css`
     position: relative;
     overflow: hidden;
   }
+  ha-card.sections {
```

Comment:
Is it not sufficient to set `height`?

Reply:
Yeah height is sufficient as seen in #1155